src/test/java/com/exasol/adapter/dialects/exasol/AbstractExasolSqlDialectIT.java

```diff
@@ -337,7 +385,8 @@ private void assertVsQuery(final String sql, final Matcher<ResultSet> expected)
         try {
             assertThat(query(sql), expected);
         } catch (final SQLException exception) {
-            fail("Unable to run assertion query: " + sql + "\nCaused by: " + exception.getMessage());
+            throw new AssertionFailedError(
+                    "Unable to run assertion query: " + sql + "\nCaused by: " + exception.getMessage(), exception);
```

Comment:
And the same here. As we will see the stack, there is no need to print the cause